--- /Users/yogapermana/Desktop/vite-vue-template/src/components/ui/empty-state/EmptyState.vue	2026-07-07 22:06:05
+++ /Users/yogapermana/Desktop/reward-loyalty-microapps-v2/src/components/ui/empty-state/EmptyState.vue	2026-05-11 11:27:17
@@ -6,9 +6,7 @@
     defineProps<{
       class?: HTMLAttributes['class']
       title?: string
-      titleClass?: HTMLAttributes['class']
       description?: string
-      descriptionClass?: HTMLAttributes['class']
       image?: string | Component
       imageSize?: 'sm' | 'md' | 'lg'
     }>(),
@@ -56,11 +54,12 @@
         v-if="title || description || $slots.default"
         class="flex max-w-xs flex-col items-center gap-2"
       >
-        <h3 v-if="title" :class="cn('text-center text-lg leading-7 font-semibold text-neutral-950', titleClass)">
+        <h3 v-if="title" class="text-center text-lg leading-7 font-semibold text-neutral-950">
           {{ title }}
         </h3>
-        <!-- eslint-disable-next-line vue/no-v-html -->
-        <p v-if="description" :class="cn('whitespace-pre-line text-center text-sm leading-5 font-normal text-neutral-700', descriptionClass)" v-html="description" />
+        <p v-if="description" class="text-center text-sm leading-5 font-normal text-neutral-700">
+          {{ description }}
+        </p>
         <slot />
       </div>
     </div>
