--- /Users/yogapermana/Desktop/vite-vue-template/src/components/ui/radio-group/index.ts	2026-07-07 22:04:08
+++ /Users/yogapermana/Desktop/reward-loyalty-microapps-v2/src/components/ui/radio-group/index.ts	2026-05-11 11:27:17
@@ -1,4 +1,3 @@
 export { default as RadioGroup } from './RadioGroup.vue'
 export { default as RadioGroupItem } from './RadioGroupItem.vue'
 export { default as RadioGroupItemLabel } from './RadioGroupItemLabel.vue'
-export { default as RadioOptionCard } from './RadioOptionCard.vue'
