--- /Users/yogapermana/Desktop/vite-vue-template/src/components/shared/bottom-sheets/confirmation/ConfirmationBottomSheet.vue	2026-05-04 13:26:54
+++ /Users/yogapermana/Desktop/reward-loyalty-microapps-v2/src/components/shared/bottom-sheets/confirmation/ConfirmationBottomSheet.vue	2026-06-05 23:22:08
@@ -13,6 +13,8 @@
     size?: 'xs' | 'sm' | 'md' | 'lg'
     /** Loading state */
     loading?: boolean
+    /** Disabled state */
+    disabled?: boolean
     /** Click handler */
     onClick: () => void
   }
@@ -95,6 +97,7 @@
           :variant="button.variant || 'primary'"
           :size="button.size || 'sm'"
           :loading="button.loading"
+          :disabled="button.disabled"
           :class="buttonLayout === 'row' ? 'flex-1' : 'w-full'"
           @click="button.onClick"
         >
