--- /Users/yogapermana/Desktop/vite-vue-template/src/components/shared/ticket-quantity-selector/index.ts	2026-07-07 22:07:35
+++ /Users/yogapermana/Desktop/reward-loyalty-microapps-v2/src/components/shared/ticket-quantity-selector/index.ts	2026-05-11 11:27:17
@@ -1,2 +1 @@
 export { default as TicketQuantitySelector } from './TicketQuantitySelector.vue'
-export { default as TicketQuantitySelectorSkeleton } from './TicketQuantitySelectorSkeleton.vue'
