--- /Users/yogapermana/Desktop/vite-vue-template/src/components/ui/checkbox/Checkbox.vue	2026-07-07 20:43:20
+++ /Users/yogapermana/Desktop/reward-loyalty-microapps-v2/src/components/ui/checkbox/Checkbox.vue	2026-05-11 11:27:17
@@ -1,8 +1,7 @@
 <script setup lang="ts">
   import type { CheckboxRootProps } from 'reka-ui'
   import type { HTMLAttributes } from 'vue'
-  import { computed } from 'vue'
-  import { Check, Minus } from 'lucide-vue-next'
+  import { Check } from 'lucide-vue-next'
   import { CheckboxIndicator, CheckboxRoot } from 'reka-ui'
   import { cn } from '@/utils/cn'
 
@@ -16,8 +15,6 @@
   const emit = defineEmits<{
     'update:checked': [value: boolean | 'indeterminate']
   }>()
-
-  const isIndeterminate = computed(() => props.checked === 'indeterminate')
 </script>
 
 <template>
@@ -33,7 +30,6 @@
         'hover:bg-primary-50 hover:border-primary-600',
         'focus-visible:border-primary-200 focus-visible:ring-primary-50 focus-visible:ring-4',
         'data-[state=checked]:bg-primary-500 data-[state=checked]:border-primary-500 data-[state=checked]:text-white',
-        'data-[state=indeterminate]:bg-primary-500 data-[state=indeterminate]:border-primary-500 data-[state=indeterminate]:text-white',
         'disabled:cursor-not-allowed disabled:border-neutral-200 disabled:bg-neutral-100 disabled:data-[state=checked]:border-neutral-200 disabled:data-[state=checked]:bg-neutral-100 disabled:data-[state=checked]:text-neutral-500',
         'aria-invalid:ring-error-500/20 aria-invalid:border-error-500',
         props.class,
@@ -45,8 +41,7 @@
       class="flex items-center justify-center text-current"
     >
       <slot v-bind="slotProps">
-        <Minus v-if="isIndeterminate" class="size-3" />
-        <Check v-else class="size-3" />
+        <Check class="size-3" />
       </slot>
     </CheckboxIndicator>
   </CheckboxRoot>
