--- /Users/yogapermana/Desktop/vite-vue-template/src/components/ui/image/Image.vue	2026-05-04 13:26:54
+++ /Users/yogapermana/Desktop/versace/src/components/ui/image/Image.vue	2026-07-05 13:59:18
@@ -102,11 +102,7 @@
   })
 
   const containerClasses = computed(() => {
-    return cn(
-      'relative overflow-hidden bg-neutral-100',
-      aspectRatioClass.value,
-      props.containerClass,
-    )
+    return cn('relative overflow-hidden', aspectRatioClass.value, props.containerClass)
   })
 
   const imageClasses = computed(() => {
